akka-http-core/src/main/scala/akka/http/javadsl/ClientTransport.scala

```diff
@@ -49,6 +47,15 @@ object ClientTransport {
   def httpsProxy(proxyAddress: InetSocketAddress): ClientTransport =
     scaladsl.ClientTransport.httpsProxy(proxyAddress).asJava
 
+  /**
+   * Returns a [[ClientTransport]] that runs all connection through the given HTTP(S) proxy using the
+   * HTTP CONNECT method.
+   *
+   * Pulls the host/port pair from the application.conf: akka.client.proxy.https.{host, port}
+   */
+  def httpsProxy()(implicit system: ActorSystem): ClientTransport =
```

Comment:
As this is the `javadsl`, implicit parameters don't really make sense. You could expect the `ActorSystem` as an explicit parameter here, or perhaps expect a `HttpsProxySettings` and let the user take care of creating it... though that would make it part of the public API (and thus need converting it from a case class to a 'regular' class because of binary compatibility)